frameworks/helloworld/tests/test_placement.py

```diff
@@ -141,7 +141,7 @@ def test_rr_by_hostname():
     }
 
     sdk_install.install(config.PACKAGE_NAME, num_private_agents * 4, additional_options=options)
-    ensure_count_per_agent(hello_count=2, world_count=2)
+    ensure_max_count_per_agent(hello_count=2, world_count=2)
```

Comment:
Doesn't round-robin also imply some ordering in the tasks? That is to say that the order of the tasks (if this is deterministic) should not differ by less than the number of agents?

Reply:
@elezar great point regarding round-robin (rr), but TBH I don't see (read: it is not inherently apparent from this test) how the configuration options specified, which are mostly a placement rule using GROUP_BY is supposed to guarantee a round-robin ordering/distribution.

Reply:
Yes, it's not really clear. For the recored, I don't think this is a blocker for merging this. But we could merge this as is.

Reply:
Merged. Please follow up with any necessary JIRAs / PRs.